checks/evaluation/signed_releases.go

```diff
@@ -55,6 +56,10 @@ func SignedReleases(name string,
 	for i := range findings {
 		f := &findings[i]
 
+		if f.Probe == releasesHaveVerifiedProvenance.Probe {
+			continue
+		}
+
```

Comment:
optional (since you probably have a follow-up PR): these can be removed

Reply:
Will leave those in since these will eventually be part of evaluation.